lib/AutoLayout.js

```diff
@@ -11,17 +11,19 @@ var getBendpoints = DiUtil.getBendpoints;
 var PADDING_NODE = 'padding_node';
 
 
-function AutoLayout() {
+function AutoLayout(config) {
```

Comment:
I suggest we don't implement config here but rather change the hard-coded values to reasonable values. If they are good enough per default, there is no need for configuration.

Reply:
Agreed, the bug would still be fixed